spark/src/main/java/org/apache/calcite/adapter/spark/SparkRules.java

```diff
@@ -57,6 +57,7 @@
 import org.apache.calcite.util.Pair;
 import org.apache.calcite.util.Util;
 
+import org.apache.commons.collections.IteratorUtils;
```

Comment:
This causes maven-dependency-plugin to fail. We don't use commons-collections in Calcite. Please use Guava classes instead. Also please run a full build and tests before submitting a patch.